dashboard/src/shared/AxiosInstance.ts

```diff
@@ -51,3 +63,8 @@ export function createAxiosInterceptors(axios: AxiosInstance, store: Store<IStor
     },
   );
 }
+
+// Error handling is added with an interceptor in index.tsx
+export const axios = Axios.create();
```

Comment:
I forgot why we don't just add it here? cc @migmartri

Reply:
I asked myself the same and I tried to add here the interceptors. What I found is that in that case we cannot mock the `store` (second parameter) in the tests so this was not testable.

Reply:
aha, thanks!